packages/api/blocks-api/src/blocks/block.ts

```diff
@@ -53,16 +53,23 @@ export declare type BlockIndexItem = {
 } & BlockIndexData;
 export declare type BlockIndex = Array<BlockIndexItem>;
 
+// TODO: use from WarningModule after the warning module is in the package (this enum is currently duplicated)
+export enum WarningSeverity {
+    critical = "critical",
+    high = "high",
+    low = "low",
+}
+
 export interface BlockWarningReport {
```

Comment:
Naming: Rename to BlockWarning

Reply:
done in: dcf6b1dd89295ccf53d7c3d65e854463f9dad43d